### PR TITLE
User record switcher

### PR DESCRIPTION
**Pull request by zuluecho9**
Liaison work. New feature for user record switcher in user menu bar. Work done: 

Added new section in 'Multiple user records doc', making that doc focus on the switcher, and not on the login message.
Linking to that from a few other related docs.

### CODE REVIEW

File: src/content/docs/accounts/accounts-billing/account-setup/multiple-logins-found.mdx [hunk 1]
```diff
@@ -1,16 +1,33 @@
...
   - Account setup
-metaDescription: "When logging into New Relic, you may see a 'multiple logins found' message and be asked to verify your email."
+metaDescription: "Details on features and messages for New Relic users with multiple user records and logins."
 redirects:
 ---
 
 import accountsLoginWithMultipleAccountsFound from 'images/accounts_screenshot-crop_login-with-multiple-accounts-found.png'
 
-If your email is associated with multiple user records, you'll see a `Multiple logins found` message at login, then you'll be asked to verify your email. This might happen if you have access to multiple New Relic organizations, or to multiple [authentication domains](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more) in an organization. 
+import accountsUserRecordSwitcher from 'images/accounts_screenshot-crop_user-record-switcher.png'
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/account-setup/multiple-logins-found.mdx:13]: @zuluecho9 In this screenshot, it's not clear what menu item leads to the sub menu thing. Maybe update the screenshot to highlight the clicking/mousing path?

File: src/content/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data.mdx [hunk 2]
```diff
@@ -40,6 +40,7 @@ Potential permissions-related factors that can affect your access:
 If you're logged in but can't find an expected account or UI page, it may be because you're in the wrong account in your organization. Some tips to troubleshoot this:
 
 * If your organization has multiple accounts and you have access to those accounts, you can switch to another account by going to the account switcher at the top left of most UI pages.
+* If you have multiple user records (for example, access to more than one New Relic organization), you can [switch between your user records](/docs/accounts/accounts-billing/account-setup/multiple-logins-found).
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data.mdx:43]: @zuluecho9 Why say "user records" here? Why not "user accounts"?

**Review by paperclypse — APPROVED** [review]
A couple minor things to look at, but otherwise looks good.

File: src/content/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data.mdx (hunk 2 above)
Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data.mdx:43]: 'account' and 'user record' is an entirely different concept. There can be multiple 'accounts' in a New Relic organization. And a user can have access to multiple accounts with the same login.

File: src/content/docs/accounts/accounts-billing/account-setup/multiple-logins-found.mdx (hunk 1 above)
Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/account-setup/multiple-logins-found.mdx:13]: i will fix that  thanks

**Pull request merged**